subsys/nrf_rpc/rpc_utils/server/crash_gen.c

```diff
@@ -13,12 +13,15 @@ static enum rpc_utils_cmd_server crash_command;
 static void crash_work_handler(struct k_work *work);
 K_WORK_DELAYABLE_DEFINE(crash_work, crash_work_handler);
 
+__attribute__((noinline, used))
 static void do_stack_overflow(void)
 {
 	volatile uint8_t arr[CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE];
```

Comment:
Consider using recurrent function call - this will ensure that stack size is always exceeded regardless of the calling thread stack size.